### PR TITLE
My posted job

### PR DESCRIPTION
**Pull request by zengni2018**
created employer published jobs page;
can see the posted jobs,also can choose jobs by id.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 *.xml
 config.php
 
+.idea/dataSources/952722df-8d9a-420e-b7c2-b7305554fb8f/storage_v2/_src_/schema/information_schema.FNRwLQ.meta
```
Comment (AxelBogos) [inline .gitignore]: Please remove this file from the commit

File: mypostedjob.php [hunk 2]
```diff
@@ -0,0 +1 @@
+<?php
```
Comment (AxelBogos) [inline mypostedjob.php]: Please remove this file from the commit

File: employer_published_jobs.php [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+<HEAD>
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.css">
+    <script src="functions.js">
+    </script>
+</HEAD>
+
+<BODY>
+<?php require 'user_dashboard_navbar.php' //nav bar
```
Comment (AxelBogos) [inline employer_published_jobs.php]: Change to employer_dashboard_navbar

File: employer_published_jobs.php [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+        <form id='jobAppliedForm' action="<?php echo htmlspecialchars($_SERVER["PHP_SELF"]); ?>" style="visibility: hidden" method="post">
+            <div class="form-group">
+                <input type="text" style="visibility: hidden" id= 'appliedJob' name="appliedJob" value="">
+                <br>
+                <input type="submit" class="btn btn-primary" value="Apply" align="left">
+            </div>
+
+        </form>
+    </td>
+</tr>
```
Comment (AxelBogos) [inline employer_published_jobs.php]: Remove this table row for now

**Review by AxelBogos — CHANGES_REQUESTED** [review]

File: employer_published_jobs.php (hunk 3 above)
Comment (zengni2018) [reply employer_published_jobs.php]: done

File: mypostedjob.php (hunk 2 above)
Comment (zengni2018) [reply mypostedjob.php]: done

File: .gitignore (hunk 1 above)
Comment (zengni2018) [reply .gitignore]: done

File: employer_published_jobs.php (hunk 4 above)
Comment (zengni2018) [reply employer_published_jobs.php]: done

**Pull request merged**